src/Components/Blazor/testassets/HostedInAspNet.Client/HostedInAspNet.Client.csproj

```diff
@@ -3,12 +3,13 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <OutputType>Exe</OutputType>
-    <ReferenceBlazorBuildLocally>true</ReferenceBlazorBuildLocally>
+    <ReferenceBlazorBuildLocally Condition="'$(BuildNodeJS)' == 'true'">true</ReferenceBlazorBuildLocally>
```

Comment:
Sorry, I'm a total n00b and I am not good with computers. What does this property do?

Reply:
I'm still debating this change. Does it make sense to build managed projects without building NodeJS projects if we need to regenerate the artifacts in Web.JS? On one hand, I think it makes sense to build Web.JS on all builds, but it does mean we are imposing a requirement to install yarn and nodejs to build our repo.

Reply:
These sound like questions that we shouldn't try to answer within the scope of a PR.